### PR TITLE
Add support for Bing and Yandex webmaster tools. Closes #147

### LINKED ISSUE #147: Add support for Bing and Yandex site verification meta tag for webmaster tools
Building on #8 we should also add support for Bing and Yandex:

```liquid
<meta name="msvalidate.01" content="{{ site.bing_site_verification }}">
<meta name="yandex-verification" content="{{ site.yandex_site_verification }}">
```

Some might also want Alexa too, but site registrations were retired in May of 2016 and authors can add those on their own.

### PR DESCRIPTION
**Pull request by jhabdas**
Adds value. Ideally would only appear on the front page as suggested by @pathawks in #45.

### CODE REVIEW

**Comment by pathawks** [comment]
Rubocop seems to be complaining that our tests are too long. :doh:

File: README.md [hunk 1]
```diff
@@ -92,6 +92,8 @@ The SEO tag will respect any of the following if included in your site's `_confi
     ```
 
 * `google_site_verification` for verifying ownership via Google webmaster tools
+* `bing_site_verification` for verifying ownership via Bing webmaster tools
+* `yandex_site_verification` for verifying ownership via Yandex webmaster tools
```
Comment (benbalter) [inline README.md]: We'll need to retain backwards compatibility with a `default` for google, but what do you think about a design pattern like:

```yml
site_verifications:
  google: XX-1234-abc
  bing: asdfasfdasf
  yandex: 12341234
```

Comment (jhabdas) [inline README.md]: I'd drop the plural form but I'm in favor of the organization. @pathawks?

Comment (pathawks) [inline README.md]: Yeah, I like this better. :+1:

**Comment by jhabdas** [comment]
Incorporated feedback and backed the webmaster verifications with some unit tests (was hoping to get a negative assertion in there but my Ruby-fu is still pretty lackluster. Travis seems to be complaining about some unrelated code style snafus elsewhere in the code. Could one of you guys take a peak at let me know how how we're looking?

**Comment by pathawks** [comment]
> Could one of you guys take a peak at let me know how we're looking?

It's not you, it's me.

**Comment by pathawks** [comment]
Tests pass now :tada:

@jekyllbot: merge +minor

**Comment by pathawks** [comment]
Thank you @jhabdas :+1:

**Comment by jhabdas** [comment]
With pleasure. Thanks for bumping the minor.

**Pull request merged**